pages/common/bird.md

```diff
@@ -0,0 +1,13 @@
+# bird
+
+> BIRD Internet Routing Daemon.
+> Routing daemon with support for BGP, OSPF, Babel and others.
+> More information: <https://bird.network.cz/>.
+
+- Start Bird with a specific configuration file:
+
+`bird -c {{bird.conf}}`
+
+- Start Bird as a specific user and group:
+
+`bird -c {{bird}} -g {{bird}}`
```

Comment:
I think command is wrong
```suggestion
`bird -u {{username}} -g {{group}}`
```

Reply:
How I came up with that is beyond me - thanks for the correction.